src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -153,6 +157,12 @@ export class MatSlideToggle extends _MatSlideToggleMixinBase implements OnDestro
   @Output() readonly change: EventEmitter<MatSlideToggleChange> =
       new EventEmitter<MatSlideToggleChange>();
 
+  /** An event will be dispatched each time the slide-toggle input is toggled. */
+  @Output() readonly toggleChange: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
Just a nit: I think it would be really appropriate to expand this comment further, to mention that this always fires, but does not mean that the slide-toggle value is always being updated. Same for the `dragChange` event.

Otherwise, I think there is the risk that people might use `toggleChange` over the usual `change` event.